src/server/src/main/java/io/cassandrareaper/jmx/ClusterFacade.java

```diff
@@ -781,6 +781,12 @@ public void clearSnapshot(String snapshotName, Node host) throws ReaperException
    * @throws IOException if parsing the JSON breaks
    */
   public static CompactionStats parseCompactionStats(String json) throws IOException {
+    if (json.isEmpty()) {
+      return CompactionStats.builder()
```

Comment:
can we have `return CompactionStats.empty();` instead please. (or `CompactionStats.undefined()`)

Reply:
Would an optional be appropriate here after adjusting the method signature?
```suggestion
      return Optional<CompactionStats>.empty()
```